rector.php

```diff
@@ -24,26 +25,20 @@
         SetList::EARLY_RETURN,
         SetList::CODING_STYLE,
         SetList::STRICT_BOOLEANS,
-    ]);
-
-    $rectorConfig->rules([DeclareStrictTypesRector::class, MoveAbstractRectorToChildrenRector::class]);
-
-    $rectorConfig->paths([
+    ])->withRules([DeclareStrictTypesRector::class, MoveAbstractRectorToChildrenRector::class])
```

Comment:
My inner linter 😅 

```suggestion
])
->withRules([DeclareStrictTypesRector::class, MoveAbstractRectorToChildrenRector::class])
```

But anyway, I wanted to say it's probably good idea to have such builder, but why it was merge without tests 😅?